### DISCUSSION TITLE
Default profile images

### DISCUSSION THREAD

**Discussion by BenGWeeks** [damus-io/damus#230] [Ideas]
How attached are we to the Robohash iconography? Not come across it before and pretty clever how it works, but wondered if we could implement Nostrichs from @yeg0rpetrov [npub1z4m7gkva6yxgvdyclc7zp0vz4ta0s2d9jh8g83w03tp5vdf3kzdsxana6p]? We're not keen on bots are we?! :-)

**Comment by olheadinternet** [comment]
I like that the bots are generated from pubkeys.

**Reply by BenGWeeks** [reply]
Me too. Perhaps could implement a very similar thing but just gets first character of nPub with 10 varieties (I know, not unique).

**Reply by n0rthranger** [reply]
I guess ostriches would make more sense rather than robots.

**Reply by olheadinternet** [reply]
robohash is open and it just copy and pastes png images together... i might play around with building an ostrich themed set when i get a doodle itch. https://github.com/e1ven/Robohash/tree/master/robohash

**Comment by BenGWeeks** [comment]
![image](https://user-images.githubusercontent.com/5950171/210368981-5f9ffc40-97f2-4ee1-a9fd-f0900e4c3628.png)

![image](https://user-images.githubusercontent.com/5950171/210369011-c181767f-73d3-45c3-b07d-e9e018fd5a0d.png)

![image](https://user-images.githubusercontent.com/5950171/210369029-3cda99f5-2f86-43d1-b6b5-ce6b727babe7.png)

**Comment by BenGWeeks** [comment]
Simple solution could be to just take first letter of hex nPub that translates to one of 16 Ostriches.
